ChromePhp.php

```diff
@@ -244,33 +244,31 @@ public static function table()
      * internal logging call
      *
      * @param string $type
-     * @return void
+     * @return ChromePhp
      */
     protected static function _log($type, array $args)
     {
+        $logger = self::getInstance();
+
         // nothing passed in, don't do anything
-        if (count($args) == 0 && $type != self::GROUP_END) {
-            return;
+        if (empty($args) && $type != self::GROUP_END) {
+            return $logger;
         }
 
-        $logger = self::getInstance();
-
         $logger->_processed = array();
-
-        $logs = array();
-        foreach ($args as $arg) {
-            $logs[] = $logger->_convert($arg);
-        }
+        $logs = array_map(array($logger, '_convert'), $args);
 
         $backtrace = debug_backtrace(false);
         $level = $logger->getSetting(self::BACKTRACE_LEVEL);
 
         $backtrace_message = 'unknown';
-        if (isset($backtrace[$level]['file']) && isset($backtrace[$level]['line'])) {
+        if (isset($backtrace[$level]['file'], $backtrace[$level]['line'])) {
```

Comment:
This change is about two styles for presenting check two values will be set.

If we accept this, we should change all related conditions to this in this respo.